packages/devextreme-cli/package.json

```diff
@@ -30,7 +30,7 @@
     "test-template": "node ./testing/testing.js",
     "test-dev": "cross-env TEST_MODE=dev jest",
     "test-themebuilder": "jest --config=./jest-themebuilder.config.js --runInBand",
-    "docker:build": "docker build --rm -f \"testing/docker/Dockerfile\" -t devextreme-cli:latest \"testing/docker\"",
+    "docker:build": "docker build --rm -f \"testing/docker/Dockerfile\" -t devextreme-cli:latest \"testing/docker\" --progress=plain",
```

Comment:
Are we sure we still need docker support?

Reply:
Please @iBat reply

Reply:
Docker is not currently used in the CI/CD workflow. But probably still needed in development.

Reply:
Since it's now paid, it makes sense to remove it completely.